Makefile

```diff
@@ -230,6 +230,14 @@ ifdef SCALABILITY_KUEUE_LOGS
 SCALABILITY_EXTRA_ARGS +=  --withLogs=true --logToFile=true
 endif
 
+ifdef SCALABILITY_SCRAPE_INTERVAL
+SCALABILITY_SCRAPE_ARGS +=  --metricsScrapeInterval=$(SCALABILITY_SCRAPE_INTERVAL)
```

Comment:
Can we add them to `SCALABILITY_EXTRA_ARGS`? I guess one could use them with `kubectl proxy` and scrape from a local kind cluster.

Reply:
`SCALABILITY_EXTRA_ARGS` is used for args that are relevant for minimalkueue usecase only.

>  I guess one could use them with kubectl proxy and scrape from a local kind cluster.
I don't understand.

Reply:
I was suggesting that we could do:
```
ifdef SCALABILITY_SCRAPE_INTERVAL
SCALABILITY_EXTRA_ARGS +=  --metricsScrapeInterval=$(SCALABILITY_SCRAPE_INTERVAL)
endif

ifdef SCALABILITY_SCRAPE_URL
SCALABILITY_EXTRA_ARGS +=  --metricsScrapeURL=$(SCALABILITY_SCRAPE_URL)
endif
```
As a consequence the params will be also passed to `run-scalability-in-cluster`, and I think this could be useful.
If I'm running a local cluster on Kind, then I can use `kubectl proxy` and pass `metricsScrapeURL=http://localhost:8001/metrics`. This will let use scrape the data from the cluster and analyze.

Reply:
Passing potential cpu profile or logging params don't make sence in a run-in-cluster scenarion, that is why we have two sets.

Regarding the scrape URL, it's not relevant how it is managed (`kubectl port-forward`  or something else) as long as it's accessable.

Reply:
> Passing potential cpu profile or logging params don't make seance in a run-in-cluster scenario, that is why we have two sets.

I see, but then they could be just ignored by the script. It feels a complication to have `SCALABILITY_EXTRA_ARGS` and `SCALABILITY_SCRAPE_ARGS` at the top level Makefile.

Reply:
which script?

Reply:
I meant the scalability runner

Reply:
Why mix them here and extract them afterwords when it's that easy to keep the separated?

Reply:
Readability only, I was a little surprised and it wasn't immediately obvious why we have both `SCALABILITY_SCRAPE_ARGS` and `SCALABILITY_EXTRA_ARGS`, because the scrape args are "extra". 

Anyway, this is not a strong opinion, I'm ok as is if this is your preference, I see the reasoning now.